com.valvesoftware.Steam.CompatibilityTool.Proton.yml

```
@@ -732,6 +746,53 @@ modules:
         sha256: 8ff2fcf9b054e4a4ef56fcd8a637322f827b2b176a592a618d63672ddb896e06
 
 
+  - name: vkd3d
+    subdir: vkd3d
+    build-options:
+      ldflags: >-
+        -L/usr/lib/sdk/mingw-w64/x86_64-w64-mingw32/lib
+        -L/app/share/steam/compatibilitytools.d/Proton-GE/lib/wine
+      ldflags-override: true
+      config-opts:
+        - --host=x86_64-w64-mingw32
+      env:
+        CC: ccache x86_64-w64-mingw32-gcc
+        LD: x86_64-w64-mingw32-ld
+      prefix: /app/share/steam/compatibilitytools.d/Proton-GE/lib/vkd3d/prefix
+    config-opts: &vkd3d-config-opts
+      - --disable-doxygen-doc
+      - --disable-tests
+      - --disable-demos
+      - --without-ncurses
+    post-install:
+      - mv -v ${FLATPAK_DEST}/lib/vkd3d/{prefix/bin/,}libvkd3d-shader-1.dll
+    sources: &vkd3d-sources
+      - *proton_source
+    cleanup:
+      - /lib/vkd3d/prefix
+
+
+  - name: vkd3d-32bit
+    subdir: vkd3d
+    build-options:
+      ldflags: >-
+        -L/usr/lib/sdk/mingw-w64/i686-w64-mingw32/lib
+        -L/app/share/steam/compatibilitytools.d/Proton-GE/lib32/wine
```

...and `lib32/wine/i386-windows` here
```suggestion
        -L/app/share/steam/compatibilitytools.d/Proton-GE/lib32/wine/i386-windows
```